tests/system/cases/datasources/main.tf

```diff
@@ -0,0 +1,61 @@
+provider "layer0" {
+  endpoint        = "${var.endpoint}"
+  token           = "${var.token}"
+  skip_ssl_verify = true
+}
+
+resource "layer0_environment" "datasources" {
+  name = "dsrctest"
+}
+
+resource "layer0_load_balancer" "datasources" {
+  name        = "dsrctest_lb"
+  environment = "${layer0_environment.datasources.id}"
+
+  port {
+    host_port      = 80
+    container_port = 80
+    protocol       = "http"
+  }
+}
+
+resource "layer0_service" "datasources" {
+  name          = "dsrctest_svc"
+  environment   = "${layer0_environment.datasources.id}"
+  deploy        = "${layer0_deploy.datasources.id}"
+  load_balancer = "${layer0_load_balancer.datasources.id}"
+  scale         = "1"
+  wait          = true
```

Comment:
As in wait is the default? I explicitly set wait to false which doesn't work as I needed data to be read after the resource creation was complete, especially when accessing service.load_balancer or load_balancer.service. So I explicitly set it to true to (despite it being the default).

But as I've removed load balancer <-> service links, I agree we no longer need the wait.
✅